assets/js/modules/analytics-4/datastore/conversion-reporting.js

```diff
@@ -268,12 +276,25 @@ export const selectors = {
 	hasLostConversionReportingEvents:
 		hasConversionReportingEventsOfType( 'lostEvents' ),
 
+	/**
+	 * Returns newBadgeEvents if present.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return {Array|undefined} `newBadgeEvents` array if events are present, `undefined` otherwise.
+	 */
+	getNewBadgeEvents: createRegistrySelector( ( select ) => () => {
+		const inlineData =
+			select( MODULES_ANALYTICS_4 ).getConversionReportingEventsChange();
+
+		return inlineData?.newBadgeEvents;
+	} ),
```

Comment:
Makes sense, although this specific selector is not going to be used in any loading context, since it is only used to determine if badge should be shown or not - without having a loader etc, I added it for consistency.

I applied suggestion directly here, since it has no effect on existing codeabse, no other change beside this one is needed